zavit.Web.Mvc/App/Modules/messaging/messageLayout.js

```diff
@@ -42,9 +43,21 @@ export function adjustHeightOfMainContainer($messagesContainer) {
         heightOfMargin = parseInt($("#controls").css("margin-top")),
         heightOfContainerMargin = parseInt($("#messagesContainer").css("margin-top")),
         heightOfFooter = $(".footer").height();
-        let heightOfMessagesContainer = $(window).height() - topOfMessages - heightOfControls - heightOfFooter - heightOfMargin - heightOfContainerMargin;
 
-        if (window.matchMedia("(max-width: 990px)").matches) heightOfMessagesContainer = heightOfMessagesContainer - parseInt($("#arrangeNew").css("margin-top"));
+    let heightOfMessagesContainer = $(window).height() - heightOfControls - heightOfFooter - heightOfMargin - heightOfContainerMargin;
+
+    if (window.matchMedia("(max-width: 990px)").matches) {
+
+        heightOfMessagesContainer = heightOfMessagesContainer -
+            $("#threadTitle").position().top -
+            $("#threadTitle").height() -
+            $("#backToInbox").height() +
+            parseInt($("#backToInbox").css("margin-bottom")) +
+            parseInt($("#threadTitle").css("margin-bottom"));
```

Comment:
could select once into variable const threadTitle = $("#threadTitle"); and then use that instead of repeating selector